test_client.py

```diff
@@ -1,20 +1,38 @@
 from client import client
-import pytest
 
 
-def test_string_input():
-    assert client("String") == "You sent: String"
-
-
-def test_int_input():
-    assert client(42) == "You sent: 42"
-
-
-def test_empty_input():
-    with pytest.raises(TypeError):
-        client()
-
-
-def test_over32_input():
-    assert client("A long message that will be over 32 bits but here's a few more")\
-    == "You sent: A long message that will be over 32 bits but here's a few more"
+def test_response_ok():
+    msg = "GET /path/to/myindex.html HTTP/1.1\r\nHost: localhost:50000\r\n"
```

Comment:
I see now why you think there are 5 parts to the HTTP request, but there is so much more that might be there.  Try requesting something via your browser and your server will choke on it.